### PR TITLE
add(mysql): connect to Aiven for MySQL with  dbeaver and datagrip

### PR DESCRIPTION
**Pull request by wojcik-dorota**
https://aiven.atlassian.net/browse/DOC-700

## Checklist

- [x] The first paragraph of the page is on one line.
- [x] The other lines have a line break at 90 characters.
- [x] I checked the output.
- [x] I applied the [style guide](../styleguide.md).
- [x] My links start with `/docs/`.

### CODE REVIEW

**Review by divdby0 — APPROVED** [review]

File: docs/products/mysql/howto/connect-with-datagrip.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+   the [Aiven Console](https://console.aiven.io/).
+1. Select **OK** to create and save the connection.
+
+The connection to your Aiven for MySQL service has been established and is visible in
+DataGrip > **Database Explorer**.
+
+## Related pages
+
+- [Connect to Aiven for MySQL](/docs/products/mysql/howto/list-code-samples) for
+more tools you can use for connecting to your service
```
Comment (ArthurFlag) [inline docs/products/mysql/howto/connect-with-datagrip.md:41]:
```suggestion
  more tools you can use for connecting to your service
```

File: docs/products/mysql/howto/connect-with-datagrip.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+---
+title: Connect to Aiven for MySQL® with DataGrip
+sidebar_label: Connect with DataGrip
+---
+
+Use [DataGrip](https://www.jetbrains.com/datagrip/) to connect to your Aiven for
+MySQL® service.
```
Comment (ArthurFlag) [inline docs/products/mysql/howto/connect-with-datagrip.md:7]:
```suggestion
Use [DataGrip](https://www.jetbrains.com/datagrip/) to connect to your Aiven for MySQL® service.
```

File: docs/products/mysql/howto/connect-with-datagrip.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+- Access to the [Aiven Console](https://console.aiven.io/)
+- At least one running Aiven for MySQL service
+- [DataGrip](https://www.jetbrains.com/datagrip/download/) installed on your machine
+
+## Get JDBC URI from Aiven Console
+
+1. Log in to [Aiven Console](https://console.aiven.io/) and navigate to your organization
+   \> project > Aiven for MySQL service.
+1. On the service **Overview** page, select **Quick connect**.
+1. In the **Connect** window
```
Comment (ArthurFlag) [inline docs/products/mysql/howto/connect-with-datagrip.md:20]:
```suggestion
1. In the **Connect** window:
```

File: docs/products/mysql/howto/connect-with-dbeaver.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+   [Aiven Console](https://console.aiven.io/).
+1. Select **Finish** to create and save the connection.
+
+The connection to your Aiven for MySQL service has been established and is visible in
+DBeaver > **Database Navigator**.
+
+## Related pages
+
+- [Connect to Aiven for MySQL](/docs/products/mysql/howto/list-code-samples) for
+more tools you can use for connecting to your service
```
Comment (ArthurFlag) [inline docs/products/mysql/howto/connect-with-dbeaver.md:42]:
```suggestion
   more tools you can use for connecting to your service.
```

File: docs/products/mysql/howto/connect-with-dbeaver.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+1. In the **Connect** window
+
+   1. Choose to connect with Java using the **Connect with**
+      dropdown menu.
+   1. Copy the generated JDBC URI.
+   1. Select **Done**.
+
+## Connect to JDBC URI from DBeaver
+
+1. Open DBeaver on your machine, and select **Database** > **New Database Connection**
```
Comment (ArthurFlag) [inline docs/products/mysql/howto/connect-with-dbeaver.md:28]:
```suggestion
1. Open DBeaver and click **Database** > **New Database Connection**
```

File: docs/products/mysql/howto/connect-with-dbeaver.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+   from the top navigation menu.
+1. In the **Connect to database** window, select MySQL and click **Next**.
+1. In the **Connection Settings** window > **Main** tab > **Server** section,
+   choose to connect with URL and paste the URI copied from the
+   [Aiven Console](https://console.aiven.io/).
+1. Select **Finish** to create and save the connection.
```
Comment (ArthurFlag) [inline docs/products/mysql/howto/connect-with-dbeaver.md:34]:
```suggestion
1. Click **Finish** to create and save the connection.
```

File: docs/products/mysql/howto/connect-with-dbeaver.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+- Access to the [Aiven Console](https://console.aiven.io/)
+- At least one running Aiven for MySQL service
+- [DBeaver](https://dbeaver.io/download/) installed on your machine
+
+## Get JDBC URI from Aiven Console
+
+1. Log in to [Aiven Console](https://console.aiven.io/) and navigate to your organization
+   \> project > Aiven for MySQL service.
+1. On the service **Overview** page, select **Quick connect**.
+1. In the **Connect** window
```
Comment (ArthurFlag) [inline docs/products/mysql/howto/connect-with-dbeaver.md:19]:
```suggestion
1. In the **Connect** window:
```

File: docs/products/mysql/howto/connect-with-datagrip.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+
+   1. Choose to connect with Java using the **Connect with**
+      dropdown menu.
+   1. Copy the generated JDBC URI.
+   1. Select **Done**.
```
Comment (ArthurFlag) [inline docs/products/mysql/howto/connect-with-datagrip.md:25]:
```suggestion
   1. Click **Done**.
```

File: docs/products/mysql/howto/connect-with-datagrip.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+
+- Access to the [Aiven Console](https://console.aiven.io/)
+- At least one running Aiven for MySQL service
+- [DataGrip](https://www.jetbrains.com/datagrip/download/) installed on your machine
+
+## Get JDBC URI from Aiven Console
+
+1. Log in to [Aiven Console](https://console.aiven.io/) and navigate to your organization
+   \> project > Aiven for MySQL service.
+1. On the service **Overview** page, select **Quick connect**.
```
Comment (ArthurFlag) [inline docs/products/mysql/howto/connect-with-datagrip.md:19]:
```suggestion
1. On the service **Overview** page, click **Quick connect**.
```

File: docs/products/mysql/howto/connect-with-dbeaver.md [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+
+- Access to the [Aiven Console](https://console.aiven.io/)
+- At least one running Aiven for MySQL service
+- [DBeaver](https://dbeaver.io/download/) installed on your machine
+
+## Get JDBC URI from Aiven Console
+
+1. Log in to [Aiven Console](https://console.aiven.io/) and navigate to your organization
+   \> project > Aiven for MySQL service.
+1. On the service **Overview** page, select **Quick connect**.
```
Comment (ArthurFlag) [inline docs/products/mysql/howto/connect-with-dbeaver.md:18]:
```suggestion
1. On the service **Overview** page, click **Quick connect**.
```

File: docs/products/mysql/howto/connect-with-dbeaver.md [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+1. In the **Connect** window
+
+   1. Choose to connect with Java using the **Connect with**
+      dropdown menu.
+   1. Copy the generated JDBC URI.
+   1. Select **Done**.
```
Comment (ArthurFlag) [inline docs/products/mysql/howto/connect-with-dbeaver.md:24]:
```suggestion
   1. Click **Done**.
```

**Review by ArthurFlag — DISMISSED** [review]
minor style guide-related fixes

File: docs/products/mysql/howto/connect-with-dbeaver.md (hunk 5 above)
Comment (wojcik-dorota) [reply docs/products/mysql/howto/connect-with-dbeaver.md:28]: Here the users actually select options from a number of other possible options available from the top navigation menu. I'd keep "select" in that case. Wdyt?

Comment (ArthurFlag) [reply docs/products/mysql/howto/connect-with-dbeaver.md:28]: Sure thing, makes sense!

**Review by ArthurFlag — APPROVED** [review]

**Pull request merged**